### PR TITLE
9901-graphql updated for new Q&A content on support resource detail pages

### PR DESCRIPTION
**Pull request by jtmst**
## Description
updated graphql query to pull new QA content

https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9901

## Testing done
Tested in PR tugboat environment: https://tugboat.vfs.va.gov/62c7e70524e7d21f96728554

https://cms-zqis1amj7i3jmmqvydvdxfmptbsn7zzf.ci.cms.va.gov/admin/content/deploy
https://web-zqis1amj7i3jmmqvydvdxfmptbsn7zzf.ci.cms.va.gov/resources/how-to-get-your-medical-records-from-your-va-health-facility/

Trigger content build and release using the frontend from this PR.  Observe that the environment builds without error

## Screenshots
Screen Shot 2022-08-15 at 10 49 54 AM

Screen Shot 2022-08-15 at 10 48 47 AM

## Acceptance criteria
- [ ] Query pulls in new Q&A content
- [ ]  Updated content model must be present in main before this can be merged

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by swirtSJW** [comment]
@jtmst can you update the name of this pr to be a bit more descriptive and reference the issue?   The title becomes the commit message and makes it a lot easier to interpret what is going on when looking at blame to try to figure out why something was done.

File: src/site/stages/build/drupal/graphql/nodeSupportResourcesDetailPage.graphql.js [hunk 1]
```diff
@@ -125,7 +154,7 @@ const GetNodeSupportResourcesDetailPage = `
       ]
     }) {
       entities {
-        ... nodeSupportResourcesDetailPage
+				... nodeSupportResourcesDetailPage
```
[the remark this answers is not in the document]
Comment (jtmst) [reply src/site/stages/build/drupal/graphql/nodeSupportResourcesDetailPage.graphql.js:157]: this is normal on my local but shows up weird in GH?  I'll see what i can do

**Review by dsasser — APPROVED** [review]

File: src/site/stages/build/drupal/graphql/nodeSupportResourcesDetailPage.graphql.js [hunk 2]
```diff
@@ -48,6 +49,7 @@ fragment nodeSupportResourcesDetailPage on NodeSupportResourcesDetailPage {
       ... contactInformation
     }
   }
+
```
Comment (ryguyk) [inline src/site/stages/build/drupal/graphql/nodeSupportResourcesDetailPage.graphql.js:52]: Probably don't need this blank line.

**Review by maxx1128 — APPROVED** [review]
Successful build for me

**Review by ryguyk — APPROVED** [review]
Looks good. One blank line that's not necessary. Fix if you see fit.

**Pull request merged**